test_client/src/main/scala/io/reactivesocket/tck/ClientDSL.scala

```diff
@@ -4,7 +4,7 @@ import java.io.{File, PrintWriter}
 
 class ClientDSL {
 
-  val writer: PrintWriter = new PrintWriter(new File(this.getClass.getSimpleName + ".txt"))
+  var writer: PrintWriter = new PrintWriter(new File(this.getClass.getSimpleName + ".txt"))
```

Comment:
Yup, no worries this can all be improved over time. Just commenting on anything that caught my eye and trying to understand the codebase now :-)